api/appStore/AppStoreRouter.go

```diff
@@ -83,4 +83,9 @@ func (router AppStoreRouterImpl) Init(configRouter *mux.Router) {
 
 	configRouter.Path("/cluster-component/install/{clusterId}").
 		HandlerFunc(router.deployRestHandler.DefaultComponentInstallation).Methods("POST")
+
+	configRouter.Path("/installed-app/deployment-history").Queries("installedAppId", "{installedAppId}").
```

Comment:
it should be of the form `installed-app/${installedAppId}/deployment-history`

Reply:
will have separate exercise where we refactor app store related urls